components/automate-ui/src/app/modules/infra-proxy/create-environment-modal/create-environment-modal.component.scss

```diff
@@ -0,0 +1,454 @@
+@import "~styles/variables";
+
+.version-dropdown {
+  color: $chef-primary-dark;
+
+  ::ng-deep label {
+    display: inline;
+    color: $chef-primary-dark;
+    font-size: 15px;
+    margin-right: 15px;
+  }
+
+  chef-select {
+    margin-bottom: 12px;
+    width: 150px;
+    font-size: 14px;
+  }
+}
+
+chef-modal {
+
+  [slot=title] {
+    text-align: left;
+    font-weight: bold;
+    padding-bottom: 44px;
+    margin-top: 0px;
+  }
+
+  .flex-container {
+    justify-content: center;
+
+    .input-margin {
+      margin-bottom: 20px;
+    }
+
+    form {
+      flex-basis: 61%;
+
+      input {
+        font-size: 1em;
+        height: 36px;
+        left: 398px;
+        right: 398px;
+        top: 365px;
+        border: 1px solid $chef-form-input;
+        box-sizing: border-box;
+        border-radius: 4px;
+        margin-top: 8px;
+        margin-top: 8px;
+      }
+
+      label {
+        padding-bottom: 0;
+        font-size: 16px;
+        line-height: 20px;
+        letter-spacing: 0.25px;
+        color: $chef-black;
+        margin-top: 32px;
+      }
+
+      #description {
+        height: 162px;
+        left: 398px;
+        right: 398px;
+        top: 462px;
+        border: 1px solid $chef-form-input;
+        box-sizing: border-box;
+        border-radius: 4px;
+        margin-top: 10px;
+      }
+
+      #default, #override {
+        height: 286px;
+      }
+
+      ::ng-deep #default::-webkit-scrollbar {
+        width: 5px;
+        background: $chef-form-input;
+        border-radius: 5px;
+      }
+
+      ::ng-deep #default::-webkit-scrollbar-thumb {
+        background: #B4C3CC;
```

Comment:
Avoid color literals -- create a named constant.

Reply:
yes i have added this to variable file